### PR TITLE
rebranding .md files

### PR DESCRIPTION
**Pull request by tudor-malene**
### Why this change is needed

Rebrand from Obscuro to Ten

### What changes were made as part of this PR

replace "Obscuro" in the .md files.

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,28 +334,28 @@ Will then start each component with the alloted defaults. ~3-6 mins to complete
...
-Deploying and interacting with contracts on Obscuro requires ETH to be allocated to an account on the L2 via the faucet (or bridged across from the L1).
+Deploying and interacting with contracts on Ten requires ETH to be allocated to an account on the L2 via the faucet (or bridged across from the L1).
 The faucet image should be pulled (or optionally built from scratch), and the container built and started to allow requests to be made to it. 
 To pull the testnet faucet and run use;
 
 ```
 cd tools/faucet
-docker pull testnetobscuronet.azurecr.io/obscuronet/faucet_sepolia_testnet:latest  
-./container_run.sh -image testnetobscuronet.azurecr.io/obscuronet/faucet_sepolia_testnet:latest 
+docker pull testnettennet.azurecr.io/tennet/faucet_sepolia_testnet:latest
```
Comment (BedrockSquirrel) [inline README.md:357]: Is this what they are going to be? Be tempted to leave these sorts of references unchanged so they can be found when they get replaced in bulk.

File: design/architecture/overall_design.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+The purpose of this document is to describe aspects of Ten's technical design that are not addressed in the 
+[Ten whitepaper](https://whitepaper.obscu.ro/).
 
 ## Overview
 
-The following diagram shows the key components of an Obscuro deployment:
+The following diagram shows the key components of an Ten deployment:
 
-![architecture diagram](./resources/obscuro_arch.jpeg)
+![architecture diagram](./resources/ten_arch.jpeg)
```
Comment (BedrockSquirrel) [inline design/architecture/overall_design.md:12]: Need to rename this diagram

File: docs/_docs/community/contributions.md [hunk 3]
```diff
@@ -22,4 +22,4 @@ Some tasks, like competitions, accept multiple submissions. If there's a suggest
 
 Finally, nothing is more important than the community. We're here to help in any way. If you need guidance, want to discuss a task, ask questions, anything then reach out to anyone on the Obsuro Labs team on [Discord](https://discord.gg/7pkKv2Tyfn).
 
-Ready to go? Begin [here](https://app.dework.xyz/o/obscuro-6jhKTFlgOKNhKW1sG9Pbpt/p/community-co-7bsAxi8yDlzjUc3BV4hKyg?inviteId=de008bf8-d8f0-4429-8433-7f4d575e89d8)
+Ready to go? Begin [here](https://app.dework.xyz/o/ten-6jhKTFlgOKNhKW1sG9Pbpt/p/community-co-7bsAxi8yDlzjUc3BV4hKyg?inviteId=de008bf8-d8f0-4429-8433-7f4d575e89d8)
```
Comment (BedrockSquirrel) [inline docs/_docs/community/contributions.md:25]: Probably want to leave this until we have sorted out dework url

File: docs/_docs/testnet/obscuroscan.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-Obscuro's equivalent of Etherscan. ObscuroScan allows you to view the latest rollups and transactions on the Testnet, 
+# TenScan
+TenScan is a blockchain explorer for the Ten Testnet - 
+Ten's equivalent of Etherscan. TenScan allows you to view the latest rollups and transactions on the Testnet, 
 as well as search for historical rollups and transactions. Its functionality will be expanded over time.
 
 ## Usage
 
-1. Go to the [ObscuroScan landing page](http://testnet.obscuroscan.io/)
+1. Go to the [TenScan landing page](http://testnet.tenscan.io/)
```
Comment (BedrockSquirrel) [inline docs/_docs/testnet/obscuroscan.md:10]: Have we bagged this URL?

File: docs/_docs/wallet-extension/wallet-extension.md [hunk 5]
```diff
@@ -75,8 +75,8 @@ tools.
    automatically by the wallet extension. Your balance in MetaMask will be set to 0 at the beginning.
    You can get testnet ETH from [faucet](https://docs.obscu.ro/testnet/faucet/) (you may need to switch to another network and back again to force MetaMask to refresh balance).
    
-   Once a viewing key is generated it will be persisted across restarts of the wallet extension, saved in the user home space under `~/.obscuro/wallet_extension_persistence`.
+   Once a viewing key is generated it will be persisted across restarts of the wallet extension, saved in the user home space under `~/.ten/wallet_extension_persistence`.
```
Comment (BedrockSquirrel) [inline docs/_docs/wallet-extension/wallet-extension.md:78]: Might want to leave this one so it can be found when it's updated but not sure

**Review by BedrockSquirrel — COMMENTED** [review]
LGTM, few to check

**Review by BedrockSquirrel — APPROVED** [review]
LGTM when you've checked those ones I commented

File: docs/_docs/wallet-extension/wallet-extension.md (hunk 5 above)
Comment (tudor-malene) [reply docs/_docs/wallet-extension/wallet-extension.md:78]: good spot

File: docs/_docs/testnet/obscuroscan.md (hunk 4 above)
Comment (tudor-malene) [reply docs/_docs/testnet/obscuroscan.md:10]: yeah

File: docs/_docs/community/contributions.md (hunk 3 above)
Comment (tudor-malene) [reply docs/_docs/community/contributions.md:25]: good spot

File: design/architecture/overall_design.md (hunk 2 above)
Comment (tudor-malene) [reply design/architecture/overall_design.md:12]: yeah. will do that in a different pass

File: README.md (hunk 1 above)
Comment (tudor-malene) [reply README.md:357]: ok. this looks odd

**Pull request merged**